lib/garage/controller_helper.rb

```diff
@@ -34,7 +34,7 @@ def resource_owner_id
     # Use this method to render 'unauthorized'.
     # Garage user may overwrite this method to response custom unauthorized response.
     # @return [Hash]
-    def unauthorized_render_options
+    def unauthorized_render_options(error: nil)
```

Comment:
I'm starting to think that it could better to write wrapper with an argument of `unauthorized_render_options`  to [garage-doorkeeper](https://github.com/cookpad/garage-doorkeeper) than this patch.

Since we are able to make outside the scope of doorkeeper and delegate garage-doorkeeper.

How you do you feel about it?